sdk/core/core-util/vitest.browser.config.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    browser: {
+      enabled: true,
+      headless: true,
+      name: "chromium",
+      provider: "playwright",
+    },
```

Comment:
I wonder, could we disable security in the browser to get around CORS for testing like what we do now with karma or should we just bail on doing that?

Reply:
There might be a way to pass those kinds of options, but do we have any SDKs currently that need that for their browser tests?

Reply:
See [here](https://github.com/search?q=repo%3AAzure%2Fazure-sdk-for-js%20--disable-web-security&type=code). IIRC, we do that to run AAD tests in the browser but perhaps we shouldn't?